src/ert/storage/local_ensemble.py

```diff
@@ -544,6 +698,17 @@ def unset_failure(
         if filename.exists():
```

Comment:
failure is scoped per realization, state map holds the `has_response` / `has_parameter`, and responses/params have unique keys/subkeys depending on the grouping, the `has_failure` would be a single bool per realization, and adding it would be possible but would also imply restructuring the json a bit. I'd prefer leaving it for a separate PR after this